start_jupyter_cm/windows.py

```diff
@@ -60,17 +72,26 @@ def add_jupyter_here():
         os.path.dirname(__file__), 'icons'))
     logos = {'qtconsole': os.path.join(logo_path, 'jupyter-qtconsole.ico'),
              'notebook': os.path.join(logo_path, 'jupyter.ico')}
+
+    # Here I need to create a script for pythonw
+    scr_fw = open(os.path.join(PYTHON_PATH, 'Scripts',
+                               'sjc-jupyter-qtconsole-script.py'), 'w')
+    scr_fw.write(QTCONSOLE_SCRIPT)
+    scr_fw.close()
+
     for env in ('qtconsole', 'notebook'):
-        if "WINPYDIR" in os.environ:
-            # Calling from WinPython
-            # Paths are relative, so we have to set the env first
-            script = os.path.join(os.environ["WINPYDIR"], "..",
-                                  WPSCRIPTS_FOLDER,
-                                  "env.bat")
-            script += " & jupyter-%s" % env
+        if env == 'notebook':
+            if "WINPYDIR" in os.environ:
+                # Calling from WinPython
+                # Paths are relative, so we have to set the env first
+                script = os.path.join(
+                    PYTHON_PATH, "..", WPSCRIPTS_FOLDER, "env.bat") + " & jupyter-notebook"
+            else:
+                script = os.path.join(
+                    PYTHON_PATH, 'Scripts', 'jupyter-notebook.exe')
         else:
-            script = os.path.join(
-                sys.prefix, 'Scripts', "jupyter-%s.exe" % env)
+            script = os.path.join(PYTHON_PATH, 'pythonw.exe') + ' ' + os.path.join(
```

Comment:
When working from WinPython we should first call the `env.bat` script in order to set the environment variables correctly. Currently you code doesn't call `env.bat`.